src/controllers/content/content.controller.ts

```diff
@@ -13,15 +13,17 @@ export class ContentController {
   @ApiHasPassThruAuth()
   @ApiOkResponse({ type: GitContents })
   @ApiImplicitQuery({ name: "ref", required: false, type: "string" })
+  @ApiImplicitQuery({ name: "recursive", required: false, type: "string" })
   @ApiOperation({ title: "Get content", operationId: "contents_get" })
   @ApiNotFoundResponse({})
   public async getContents(
     @Param("remote") remote: string,
     @Param("path") path: string | undefined,
     @Query("ref") ref: string | undefined,
+    @Query("recursive") recursive: string | undefined,
     @Auth() auth: RepoAuth,
   ) {
-    const content = await this.contentService.getContents(remote, path, ref, { auth });
+    const content = await this.contentService.getContents(remote, path, ref, Boolean(recursive), true, { auth });
```

Comment:
is that a safe one here `Boolean(recursive)` what option does it cover?

Reply:
I'm going to go with no
![image](https://user-images.githubusercontent.com/1031227/60827877-05f21c80-a166-11e9-9290-49a260e74239.png)

Reply:
Oo TIL so.. eval(recursive)? 😄

Reply:
haha